Gemfile

```diff
@@ -16,4 +16,6 @@ gem "concurrent-ruby"
 
 gem 'os'
 
-gem 'rmagick', '~> 6.0', '>= 6.0.1'
\ No newline at end of file
+gem 'rmagick', '~> 6.0', '>= 6.0.1'
+
+gem 'debug'
```

Comment:
This needs double checking on windows

Reply:
I don't run any Windows machines, so happy to default to `pry` since it's already installed to unblock this PR for now 👍